sdks/python/apache_beam/testing/benchmarks/cloudml/criteo_tft/criteo_test.py

```diff
@@ -0,0 +1,45 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import unittest
+import pytest
+import numpy as np
+from .criteo import fill_in_missing
+try:
+  import tensorflow_transform as tft
+  import tensorflow as tf
```

Comment:
You need to move `from .criteo import fill_in_missing` into the try except. Also, please use absolute imports `apache_beam.testing.benchmarks.cloudml.criteo_tft.criteo`